streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/HighAvailabilityTaskAssignorTest.java

```diff
@@ -71,7 +67,23 @@
 import static org.hamcrest.Matchers.not;
 import static org.junit.Assert.fail;
 
-public class HighAvailabilityTaskAssignorTest {    
+public class HighAvailabilityTaskAssignorTest {
+    private final AssignmentConfigs configWithoutStandbys = new AssignmentConfigs(
+        /*acceptableRecoveryLag*/ 100L,
+        /*maxWarmupReplicas*/ 2,
+        /*numStandbyReplicas*/ 0,
+        /*probingRebalanceIntervalMs*/ 60 * 1000L,
+        /*rackAwareAssignmentTags*/ EMPTY_RACK_AWARE_ASSIGNMENT_TAGS
+    );
+
+    private final AssignmentConfigs configWithStandbys = new AssignmentConfigs(
```

Comment:
As a side note, I think these two inner classes would benefit from using the `Builder` pattern. While the comments are useful here, it's hard to discern what the different parameters are when looking at the individual tests.  I'm not suggesting this be done in this PR but in a follow-up one if you agree.